graylog2-web-interface/src/components/permissions/GranteesList.tsx

```diff
@@ -120,7 +121,7 @@ const GranteesList = ({ activeShares, onDelete, onCapabilityChange, entityType,
           })}
         </List>
       ) : (
-        <Alert>This {entityType} has no collaborators.</Alert>
+        <Alert>This {entityTypeTitle || entityType} has no collaborators.</Alert>
```

Comment:
Not a blocker for this PR, but this is a good example for when we usually use the nullish coalescing operator.